testutil/daemon/daemon.go

```diff
@@ -274,6 +277,7 @@ func (d *Daemon) NewClientT(t testing.TB, extraOpts ...client.Opt) *client.Clien
 
 	c, err := d.NewClient(extraOpts...)
 	assert.NilError(t, err, "[%s] could not create daemon client", d.id)
+	t.Cleanup(func() { c.Close() })
```

Comment:
DOH! never mind; the cleanup was included in this PR.. 🤦 

So I guess we're good